### ISSUE TITLE
Error importing devices

### ISSUE THREAD

**Issue by alarido** [digitalocean/netbox#306]
I get this error when I'm importing a device, no other error, no details:

![image](https://cloud.githubusercontent.com/assets/1955581/16874428/86fc3814-4a99-11e6-9636-b37e2d155d14.png)

**Comment by alarido** [comment]
It happens when you do not specify the Position and Face which are supposed to be optional.

BTW, thanks for netbox it's an awesome piece of software

**Comment by jeremystretch** [comment]
We're going to need a lot more information to help you. Could you please indicate the version of software you're running and provide the CSV data you're attempting to import?

**Comment by alarido** [comment]
Sorry, I forgot.

Version 1.2.2, the CSV lines I tried:

Success:
devicename,devicetype,,serialnumber,sitename,rackname,deviceposition,devicefacingside

Failure:
devicename,devicetype,,serialnumber,sitename,rackname,,


Problem is devices I'm trying to import are non-racked I can't specify a position or facing side.


Thx in advance

**Comment by bellwood** [comment]
#198 should cover this by allowing devices not in a rack (storage, work bench, etc)

**Comment by jeremystretch** [comment]
Rack position and face are optional fields. However, the format

```
devicename,devicetype,,serialnumber,sitename,rackname,,
```

is incorrect. The correct format is

```
name,device_role,manufacturer,model,platform,serial,site,rack,position,face
```

**Comment by alarido** [comment]
Sorry I was in a hurry and I replied fast and dirty so I missed the role and manufacturer fields

I'm correctly creating the CSV when I import just one line to test and I specify the face it works:

Success:
name,device_role,manufacturer,model,platform,serial,site,rack,position,face
name,device_role,manufacturer,model,platform,serial,site,rack,,face
Failure:
name,device_role,manufacturer,model,platform,serial,site,rack,position,
name,device_role,manufacturer,model,platform,serial,site,rack,,

**Comment by jeremystretch** [comment]
Can confirm this behavior in the v1.2.2 release, but not on the current develop branch. Looks like it was fixed yesterday as part of dd62caf2f0cda5cb4a049604adbd9c71aae99f20.

Going to mark this closed, but please ask to re-open if you continue having problems with the next release.

**Comment by alarido** [comment]
Confirmed It's fixed in last stable version 1.3.0
Thx
